front/app/containers/Admin/formBuilder/components/FormBuilderSettings/PageSettings/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+// components
+import InputMultilocWithLocaleSwitcher from 'components/HookForm/InputMultilocWithLocaleSwitcher';
+import { SectionField } from 'components/admin/Section';
+import QuillMultilocWithLocaleSwitcher from 'components/HookForm/QuillMultilocWithLocaleSwitcher';
+
+// types
+import { IFlatCustomFieldWithIndex } from 'services/formCustomFields';
+
+const PageSettings = (field: IFlatCustomFieldWithIndex) => (
+  <>
+    <SectionField>
+      <InputMultilocWithLocaleSwitcher
+        id="e2e-title-multiloc"
+        name={`customFields.${field.index}.title_multiloc`}
+        label={'Title (optional)'}
```

Comment:
I think you forgot to use a translatable string here and on line `24`.

Reply:
Yep - thanks for catching that!!